.circleci/config.yml

```diff
@@ -751,8 +751,8 @@ jobs:
                       # Update maven dependencies versions from properties
                       mvn -Duser.home=${HOME} -s /tmp/.gravitee.settings.xml -B -U versions:update-properties -Dmaven.version.rules.serverId=artifactory-gravitee -Dincludes="io.gravitee.*:*" -Dexcludes="io.gravitee.policy:*,io.gravitee.connector:*,io.gravitee.resource:*,io.gravitee.service:*,io.gravitee.fetcher:*,io.gravitee.tracer:*,io.gravitee.repository:*,io.gravitee.reporter:*,io.gravitee.cockpit:*,io.gravitee.discovery:*,io.gravitee.notifier:*,com.graviteesource.notifier:*,com.graviteesource.policy:*" -DallowMajorUpdates=false -DallowMinorUpdates=false -DallowIncrementalUpdates=true -DgenerateBackupPoms=false -T 2C -no-transfer-progress
 
-                      # Remove `-SNAPSHOT` from versions
-                      mvn -Duser.home=${HOME} -s /tmp/.gravitee.settings.xml -B versions:set -DremoveSnapshot=true -DgenerateBackupPoms=false --no-transfer-progress
+                      # Remove `-SNAPSHOT` from version                    
+                      mvn versions:set-property -Dproperty=changelist -DnewVersion=
```

Comment:
I would have removed theses lines, and add `-Dchangelist=` line 761 to the `mvn clean deploy` command.
Like the example [here](https://maven.apache.org/maven-ci-friendly.html#single-project-setup)

![image](https://user-images.githubusercontent.com/13161768/173916459-27d54229-fa50-4a9c-af9d-7e9d8faf6863.png)

Reply:
Great, it was what I did first and then moved to the current changes because I wasn't sure as it will mean an empty commit for the release (To check deeper tho'). If we both think we should go like this then 🚀